### PR TITLE
Upgrade akro to version 0.0.5

### PR DESCRIPTION
**Pull request by krzentner**

### CODE REVIEW

**Comment by krzentner** [comment]
This fixes #732

**Comment by krzentner** [comment]
Running the tests locally shows that actually we'll need an akro version bump before this change.

Also, akro needs more tests.

File: src/garage/tf/policies/categorical_conv_policy.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-from akro.tf import Discrete
+from akro import Discrete
```
Comment (ryanjulian) [inline src/garage/tf/policies/categorical_conv_policy.py]: can you use `import akro` instead?

File: src/garage/tf/policies/categorical_conv_policy_with_model.py [hunk 2]
```diff
@@ -1,5 +1,5 @@
 """CategoricalConvPolicy with model."""
-from akro.tf import Discrete
+from akro import Discrete
```
Comment (ryanjulian) [inline src/garage/tf/policies/categorical_conv_policy_with_model.py]: `import akro`

File: src/garage/tf/policies/categorical_gru_policy.py [hunk 3]
```diff
@@ -1,4 +1,4 @@
-from akro.tf import Discrete
+from akro import Discrete
```
Comment (ryanjulian) [inline src/garage/tf/policies/categorical_gru_policy.py]: `import akro`

File: src/garage/tf/policies/categorical_gru_policy_with_model.py [hunk 4]
```diff
@@ -1,5 +1,5 @@
 """CategoricalGRUPolicy with model."""
-from akro.tf import Discrete
+from akro import Discrete
```
Comment (ryanjulian) [inline src/garage/tf/policies/categorical_gru_policy_with_model.py]: `import akro`

File: src/garage/tf/policies/categorical_lstm_policy.py [hunk 5]
```diff
@@ -1,4 +1,4 @@
-from akro.tf import Discrete
+from akro import Discrete
```
Comment (ryanjulian) [inline src/garage/tf/policies/categorical_lstm_policy.py]: `import akro`

File: src/garage/tf/policies/categorical_lstm_policy_with_model.py [hunk 6]
```diff
@@ -1,5 +1,5 @@
 """CategoricalLSTMPolicy with model."""
-from akro.tf import Discrete
+from akro import Discrete
```
Comment (ryanjulian) [inline src/garage/tf/policies/categorical_lstm_policy_with_model.py]: `import akro`

File: src/garage/tf/policies/categorical_mlp_policy.py [hunk 7]
```diff
@@ -1,4 +1,4 @@
-from akro.tf import Discrete
+from akro import Discrete
```
Comment (ryanjulian) [inline src/garage/tf/policies/categorical_mlp_policy.py]: `import akro`

File: src/garage/tf/policies/categorical_mlp_policy_with_model.py [hunk 8]
```diff
@@ -1,5 +1,5 @@
 """CategoricalMLPPolicy with model."""
-from akro.tf import Discrete
+from akro import Discrete
```
Comment (ryanjulian) [inline src/garage/tf/policies/categorical_mlp_policy_with_model.py]: `import akro`

File: src/garage/tf/policies/continuous_mlp_policy.py [hunk 9]
```diff
@@ -5,12 +5,12 @@
 algorithms. It accepts an observation of the environment and predicts an
 action.
 """
-from akro.tf import Box
+from akro import Box
```
Comment (ryanjulian) [inline src/garage/tf/policies/continuous_mlp_policy.py]: `import akro`

File: src/garage/tf/policies/deterministic_mlp_policy.py [hunk 10]
```diff
@@ -1,4 +1,4 @@
-from akro.tf import Box
+from akro import Box
```
Comment (ryanjulian) [inline src/garage/tf/policies/deterministic_mlp_policy.py]: `import akro`

File: src/garage/tf/policies/discrete_qf_derived_policy.py [hunk 11]
```diff
@@ -3,7 +3,7 @@
 
 This policy chooses the action that yields to the largest q-value.
 """
-from akro.tf import Discrete
+from akro import Discrete
```
Comment (ryanjulian) [inline src/garage/tf/policies/discrete_qf_derived_policy.py]: `import akro`

File: src/garage/tf/policies/gaussian_gru_policy.py [hunk 12]
```diff
@@ -1,4 +1,4 @@
-from akro.tf import Box
+from akro import Box
```
Comment (ryanjulian) [inline src/garage/tf/policies/gaussian_gru_policy.py]: `import akro`

File: src/garage/tf/policies/gaussian_lstm_policy.py [hunk 13]
```diff
@@ -1,4 +1,4 @@
-from akro.tf import Box
+from akro import Box
```
Comment (ryanjulian) [inline src/garage/tf/policies/gaussian_lstm_policy.py]: `import akro`

File: src/garage/tf/policies/gaussian_mlp_policy.py [hunk 14]
```diff
@@ -1,4 +1,4 @@
-from akro.tf import Box
+from akro import Box
```
Comment (ryanjulian) [inline src/garage/tf/policies/gaussian_mlp_policy.py]: `import akro`

File: src/garage/tf/policies/gaussian_mlp_policy_with_model.py [hunk 15]
```diff
@@ -1,5 +1,5 @@
 """GaussianMLPPolicy with GaussianMLPModel."""
-from akro.tf import Box
+from akro import Box
```
Comment (ryanjulian) [inline src/garage/tf/policies/gaussian_mlp_policy_with_model.py]: `import akro`

**Comment by krzentner** [comment]
BTW, don't merge this. I'm working on akro 0.0.4, which should fix the test
failures this PR causes.

**Comment by krzentner** [comment]
Okay, this should be good to land, unless the CI finds something I missed.

File: docs/user/implement_env.rst [hunk 16]
```diff
@@ -70,7 +70,7 @@ the base environment and add some imports:
 
     from garage.envs.base import Env
     from garage.envs.base import Step
-    from akro import Box
+    import akro
```
Comment (ryanjulian) [inline docs/user/implement_env.rst]: plz update these imports to conform the CONTRIBUTING.md (import groups)

**Comment by krzentner** [comment]
Unless the CI finds a surprising breakage, I think this PR is ready to be merged.

**Pull request merged**